### ISSUE TITLE
[iOS] "Subscribe now" fails silently

### ISSUE THREAD

**Issue by htwyford** [mozilla-mobile/mozilla-vpn-client#268]
I've been unable to subscribe in the iOS app since I downloaded it. Please see this video for STR:

https://i.imgur.com/sq7fRZM.mp4

I'm on Build 2.202011201408.

**Comment by bakulf** [comment]
This should be fixed in the latest build.

**Comment by ValentinaPC** [comment]
Verified as fixed on iOS 14.2, while using the VPN cross-plat build v. 2.202011241205.
User is able to subscribe with success and can activate the VPN app.

**Comment by htwyford** [comment]
I still can't subscribe in the iOS app. Instead of failing silently, I now get a red banner at the bottom that says "Subscription failed. Try again"

**Comment by ValentinaPC** [comment]
I've checked this today (12/2/2020), using the latest VPN cross-plat build - v. 2.202030111207 and I was able to reproduce the subscription fail, intermittently, without doing something specific.
Third time I clicked the subscribe now button was a charm. First two, returned below error.
![IMG_20201202_151037](https://user-images.githubusercontent.com/15685960/100877068-f12bcc80-34b0-11eb-93ff-16645ce74e66.jpg)
